### DISCUSSION TITLE
Grouped fields [May 13]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5690] [Discord Threads]
> 🙋‍♂️ **Aymeric**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### we want to include it in NocoDB. But its not in immediate roadmap;  
#  
> 🙋‍♂️ **Aymeric**  
#### thank you  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1095667873881538611)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
